.editorconfig

```diff
@@ -1,26 +1,33 @@
 root = true
 
-[*.cs]
+# Don't use tabs for indentation.
+[*]
```

Comment:
I don't think we should have a `[*]` section on this document, as some editors will automatically apply these settings to loaded files. Beyond that, i don't think the rules in this section can be made universal in this repo: we use CRLF line endings in some baseline files and (I believe) tabs in .sh files.